Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
+MAINTAINER Lukasz Gornicki <derberg@wp.pl>
+# add bash and git
+RUN apk update && apk add --no-cache bash git openssh && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
+# clone and install skeleton
+RUN git clone https://github.com/YaaS/docpad-skeleton-apidocs.git
```

Comment:
why cloning from github instead copy files?
```docker
COPY . /docpad-skeleton-apidocs/
```
thanks to this:
- faster build
- forks don't have to change `Dockerfile`